src/entity.rs

```diff
@@ -1619,10 +1726,267 @@ impl Entity {
             pairs.push(CodePair::new_string(5, &handle.as_string()));
         }
     }
+
+    fn push_boundary_edge(pairs: &mut Vec<CodePair>, edge: &BoundaryPathEdge) {
+        if let crate::BoundaryPathEdge::Polyline { vertex } = edge {
+            pairs.push(CodePair::new_f64(10, vertex.x));
+            pairs.push(CodePair::new_f64(20, vertex.y));
+            pairs.push(CodePair::new_f64(42, vertex.z));
+        }
+    }
+
+    fn add_custom_code_pairs_hatch(
+        pairs: &mut Vec<CodePair>,
+        hatch: &Hatch,
+        _version: AcadVersion,
+    ) {
+        // Write hatch subclass marker
+        pairs.push(CodePair::new_str(100, "AcDbHatch"));
+
+        // Write basic hatch properties
+        if hatch.elevation != 0.0 {
+            pairs.push(CodePair::new_f64(30, hatch.elevation));
```

Comment:
[The spec](https://help.autodesk.com/view/OARX/2020/ENU/?guid=GUID-C6C71CED-CE0F-4184-82A5-07AD6241F15B) states that all three `10`, `20`, and `30` codes are required for this value, even though the `10` and `20` codes will always be `0`.